drivers/gpio/gpio_shell.c

```diff
@@ -221,6 +222,7 @@ static int get_sh_gpio(const struct shell *sh, char **argv, struct sh_gpio *gpio
 static int cmd_gpio_conf(const struct shell *sh, size_t argc, char **argv, void *data)
 {
 	gpio_flags_t flags = 0;
+	long vendor_specific;
```

Comment:
Why are you using `long` instead of `gpio_flags_t` here?

Reply:
Fixed